tests/nfs/pynfs_failed.pm

```diff
@@ -14,13 +14,21 @@ use Data::Dumper;
 sub run {
     my ($self, $args) = @_;
     my $data = $args->{data};
+    my $msg;
 
     record_info('INFO', "name: $data->{name}\nclassname: $data->{classname}\ntime: $data->{time}\n");
 
     die 'failure does not exist' unless (exists($data->{failure}));
 
-    $self->{result} = 'fail';
-    record_info('ERROR', "$data->{failure}->{message}\n\n$data->{failure}->{err}\n");
+    $msg = "$data->{failure}->{message}\n\n$data->{failure}->{err}";
+
+    if ($data->{code} eq 'LOCK24' && $data->{failure}->{message} eq
+        'OP_LOCK should return NFS4_OK, instead got NFS4ERR_BAD_SEQID') {
+        $self->record_soft_failure_result("LOCK24 failure is known, verriding to softfail: bsc#1192211\n\n" . $msg);
+    } else {
```

Comment:
I this approach in tuned  https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/f1f28df67f1588f50f5a8c2529a908343e3e2f9b/tests/kernel/tuned.pm#L29 . Couldn't it be better ? There is only one fail for SP4 but for older there is at least one more.